### ISSUE TITLE
Incorrectly applied azure config parameters 

### ISSUE THREAD

**Issue by iAndriy** [hackathon-cli-recommendation/cli-recommendation#13]
[AndriyFeedback]
Thanks for building interesting az cli extension which enables more easy workflow.
I've found that during processing of positional arguments of `az next` suggested command resulted in the invalid azure config file. Passing `defaults.location coew.no_coloer$(echo true)` resulted in the `location = westus2 coew.no_color=$(echo true)` line in config json file. My expectation was to set location (value: westus2) and no_color (value: true) values. See attached screenshots for described issues below.
![Incorrectly applied config parameters 1 - A](https://user-images.githubusercontent.com/3365831/111131902-68c5c900-8581-11eb-8e1e-2f9cde8c8275.png)
![Incorrectly applied config parameters - 1 - B](https://user-images.githubusercontent.com/3365831/111131912-6a8f8c80-8581-11eb-84c2-f75982429bfd.png)
![Incorrectly applied config parameters 1 - C](https://user-images.githubusercontent.com/3365831/111131915-6b282300-8581-11eb-9c3a-3bc5e10a80bf.png)

**Comment by zhoxing-ms** [comment]
Good catch! Thanks for your feedback, I have submitted a PR [link](https://github.com/Azure/azure-cli-extensions/pull/3171) to fix this problem
